epochStart/metachain/validators_test.go

```diff
@@ -105,6 +106,7 @@ func verifyMiniBlocks(bl *block.MiniBlock, infos []*state.ValidatorInfo, marshal
 		bl.ReceiverShardID != core.AllShardId ||
 		len(bl.TxHashes) == 0 ||
 		bl.Type != block.PeerBlock {
+		log.Warn("verifyMiniBlocks existed on first if")
```

Comment:
change log message. "first if" doesn't tell that much.. maybe "miniblock is not of PeerBlock type"

Reply:
removed. It was just for testing purposes